### ISSUE TITLE
set_TextType result is an empty element

### ISSUE THREAD

**Issue by twhiteaker** [ropensci/EML#254]
set_TextType in the EML package (v.1.0.3) is returning an empty string when I specify a Word document or text file with some text in it. I think the EML package may not be finding the file correctly.

```
> file.exists("Abstract.1001.docx")
[1] TRUE
> a <- set_TextType(file = "Abstract.1001.docx")
> a
<TextType/>
> f <- system.file("Abstract.1001.docx", package = "EML")
> f
[1] ""
```

R version 3.5.1 (2018-07-02)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows >= 8 x64 (build 9200)

**Comment by cboettig** [comment]
Thanks, can you try on the current version from GitHub, (v 1.99).   (I think this was fixed in #231)

**Comment by twhiteaker** [comment]
Fixed in v 1.99. Thanks.
